medusa/__main__.py

```diff
@@ -95,6 +95,34 @@
 logger = logging.getLogger(__name__)
 
 
+def fix_incorrect_list_values(data):
```

Comment:
I would put a method in medusa/config.py. 
And just fix it for the suspected fields there.
Then call the method from main somewhere here:
https://github.com/pymedusa/Medusa/blob/release/hotfix-0.2.10/medusa/__main__.py#L1119

That way we don't clutter __main__.py more then necessary?

Reply:
I thought since we are removing this in a future version it would be best to put it all in one place?

Reply:
It's technically not a config migration. But it's easy to reuse the self.config_obj property. So maybe just add a function there named fix_config() ?

Reply:
Yeah it's really good either way